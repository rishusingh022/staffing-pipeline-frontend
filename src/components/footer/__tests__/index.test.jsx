import React from 'react';
import Footer from '../index';
import { render } from '@testing-library/react';

describe('footer', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
