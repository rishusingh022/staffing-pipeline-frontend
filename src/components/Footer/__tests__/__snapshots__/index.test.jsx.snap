// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer should render correctly 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <img
      alt="McKinsey and Company"
      class="footer-img"
      src="mckinsey-logo.png"
    />
  </div>
</DocumentFragment>
`;
